workflow-templates/jitpack_build.properties.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Trigger JitPack Build",
+  "description": "A workflow to force JitPack to build when a tag is pushed.",
+  "iconName": "hmcts-logotype-crest",
+  "categories": [
+      "Shell"
```

is this the right category? shouldn't it be

```suggestion
      "Java",
      "Deployment"
```